src/rules/add-metadata-to-token.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @title User metadata for Cumul.io
+ * @overview Add Cumul.io user metaadata to token to be used for Cumul.io dashboard filtering
+ * @gallery true
+ * @category marketplace
+ */
+
+
+function addMetadataToToken(user, context, callback) {
+  const namespace = 'https://cumulio/';
+  user.user_metadata = user.user_metadata || {};
+  user.user_metadata.cumulio = user.user_metadata.cumulio || {};
```

Comment:
```suggestion
  const cumulioMetadata = user.user_metadata.cumulio || {};
```

A little less verbose throughout but not a showstopper.